File: app/src/test/java/com/commit451/gitlab/NullTypeConverter.java
package com.commit451.gitlab;

import com.bluelinelabs.logansquare.typeconverters.StringBasedTypeConverter;

/**
 * The worst type converter you will ever see. Only used in testing if Robolectric decides to break
 * and not know what a {@link android.net.Uri} or other class is
 */
public class NullTypeConverter extends StringBasedTypeConverter {

    @Override
    public String convertToString(Object object) {
        return null;
    }

    @Override
    public Object getFromString(String string) {
        return null;
    }
}

File: app/src/test/java/com/commit451/gitlab/NullTypeConverter.kt
package com.commit451.gitlab

import com.bluelinelabs.logansquare.typeconverters.StringBasedTypeConverter

/**
 * The worst type converter you will ever see. Only used in testing if Robolectric decides to break
 * and not know what a [android.net.Uri] or other class is
 */
class NullTypeConverter : StringBasedTypeConverter<Any>() {

    override fun convertToString(`object`: Any): String? {
        return null
    }

    override fun getFromString(string: String): Any? {
        return null
    }
}
